### ISSUE TITLE
[JAVA-RESTEASY] generated codes with serializableModel=true classes have no servialVersionId

### ISSUE THREAD

**Issue by magictortoise** [swagger-api/swagger-codegen#5958]
Hi, I'm trying to generate an API with swagger-codegen-maven-plugin and set `serializableModel=true`

```
<plugin>
	<groupId>io.swagger</groupId>
	<artifactId>swagger-codegen-maven-plugin</artifactId>
	<version>2.2.2</version>
	<executions>
		<execution>
			<id>test-api</id>
			<goals>
				<goal>generate</goal>
			</goals>
			<configuration>
				<inputSpec>src/main/resources/echo.yaml</inputSpec>
				<language>jaxrs-resteasy</language>
				<output>${project.basedir}</output>
				<apiPackage>${swagger-package}.api</apiPackage>
				<modelPackage>${swagger-package}.model</modelPackage>
				<invokerPackage>${swagger-package}.handler</invokerPackage>
				<environmentVariables>							
					<hideGenerationTimestamp />
				</environmentVariables>
				<configOptions>
					<sourceFolder>src/gen/java</sourceFolder>
					<serializableModel>true</serializableModel>
				</configOptions>
			</configuration>
		</execution>
	</executions>
</plugin>
```

The generated code doesn't include 
`private static final long serialVersionUID = 1L`

Please refer to [#4311](https://github.com/swagger-api/swagger-codegen/pull/4311)

Fix should be made in modules/swagger-codegen/src/main/resources/JavaJaxRS/resteasy/pojo.mustache

**Comment by wing328** [comment]
@magictortoise are you using the latest master?
```
public class {{classname}} {{#parent}}extends {{{parent}}}{{/parent}} {{#serializableModel}}implements Serializable{{/serializableModel}} {
```
Ref: https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/resources/JavaJaxRS/resteasy/pojo.mustache#L5

**Comment by magictortoise** [comment]
@wing328 I use version 2.2.2 and I do see `implements Serializable` in POJO class. What missing is 
`private static final long serialVersionUID = 1L`

For performance and coding standard, I think we better add that line. I do see serialVersionUID is added on Java and JavaSpring category, but not in JavaJaxRS.

**Comment by wing328** [comment]
@magictortoise ah ok. Would you have time to contribute a PR to add that line?

**Comment by magictortoise** [comment]
@wing328 Sure. I will need to take a look the contribution rule then.

**Comment by wing328** [comment]
Closed via #6108
